tests/data/two_residues_same_resnum.gro

```diff
@@ -0,0 +1,196 @@
+Generated with MDTraj, t= 199.0
```

Comment:
This is the gro file of CB8-G3 of SAMPL6 challenge, I used this one for the unit test because it is the one on which I first noticed the issue